docs/docs/spec.md

```diff
@@ -631,7 +631,7 @@ The same-type relation is *not* defined to be reflexive or transitive.
     > having to cover that case.)
 
 > A further level of complexity in all this is `UNSPECIFIED`. For example, in
-> the [all-worlds] version of the following rules, a type with nullness operator
+> the [some-world] version of the following rules, a type with nullness operator
```

Comment:
`some-world` does make sense to me. It would be nice to add something like the two examples from the PR description `null-inclusive ("You can put null into it") and null-exclusive ("You can dereference it").`

Reply:
Sounds good. How do you feel about the existing non-normative comment at the end of the [null-exclusive](https://jspecify.dev/docs/spec/#null-exclusive-under-every-parameterization) section?
- Is it a clear description of the dereferencing use case?
- Does it work well for it to be at the _end_ of the section instead of the beginning?
- Would the comment fit better in some other section entirely?